distrho/src/DistrhoPluginLV2export.cpp

```diff
@@ -324,6 +325,9 @@ void lv2_generate_ttl(const char* const basename)
         pluginString += "@prefix mod:  <http://moddevices.com/ns/mod#> .\n";
 #endif
         pluginString += "@prefix opts: <" LV2_OPTIONS_PREFIX "> .\n";
+#if DISTRHO_PLUGIN_WANT_PORT_GROUPS
+        pluginString += "@prefix pg:   <" LV2_PORT_GROUPS_PREFIX "> .\n";
```

Comment:
Minor niggle: the indentation of the URLS increase by one space in the current Git version of DPF